src/js/module/dashboard/ListIssuesWAQ.js

```diff
@@ -97,43 +122,49 @@ class ListIssuesWAQ extends React.Component {
                     </div>
                 ) : (
                     <>
-                        <Title text={`Older than 4 Weeks (${_.size(issuesOlderThanFourWeeks)})`} />
+                        <Title text={`Older than 4 Weeks (${_.size(_.filter(issuesOlderThanFourWeeks, this.shouldDisplayIssue))})`} />
```

Comment:
I think it would be better to do the filtering up above at the top of the render method where the `switch` statement is. That will mean that this stuff down here doesn't need to change at all.